ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1,4 +1,33 @@
 periodics:
+- agent: kubernetes
+  cluster: build01
+  cron: 0 1 * * 1
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  labels:
+    ci.openshift.io/role: infra
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  max_concurrency: 1
+  name: periodic-build-origin-release-images
+  spec:
+    containers:
+    - args:
+      - -c
+      - |
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+        oc get bc -n ci -l app=origin-release -o custom-columns=":metadata.name" --no-headers | while read bc; do oc -n ci start-build ${bc} --wait=true; done
```

Comment:
This means that:

 - we run builds serially
 - one failed build stops the job
 - we run the job once a week

This will make for poor UX if a build fails - other builds will not be triggered that week.

Reply:
We could perhaps use scheduled builds on the BuildConfig and then only check that the last build is not failing in this job?

Reply:
Or have a 1:1 mapping from ProwJob to BuildConfig

Reply:
> Or have a 1:1 mapping from ProwJob to BuildConfig

I will follow this.

Reply:
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/15230/rehearse-15230-periodic-build-origin-release-image-origin-release-golang-1.11/1354516585976434688

Rehearsed one of the jobs.
Working...